geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionStats.java

```diff
@@ -425,7 +425,13 @@
                 false),
 
             f.createLongGauge("localMaxMemory",
-                "local max memory in bytes for this region on this member", "bytes")
+                "local max memory in bytes for this region on this member", "bytes"),
+            f.createLongCounter("partitionedRegionClearLocalDuration",
```

Comment:
these should be removed since they are for PR clear

Reply:
I do believe I did with the latest checkin.